chaos.spec.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Should this be in the other PR?

Reply:
I don't think so, no.  The point is that the bot can define some parameters, such as image location and resources.

Reply:
But knowing the resources to attach seems firmly the job of orchestrating the simulation deployment

Reply:
Hypothetically, it seems like the bot developer might know what resources it needs, the image location, and how many instances can run at once in a simulation.  There might be other config we might want to introduce in this file, but I don't have anything in particular in mind.

Practically speaking though, the dev and orchestrator are currently the same people.  And this does add complexity around the loading of this file.  Though that's more or less been implemented, with a few open questions.  We could just set a docker image URI/tag the system resources in the DB though.  I don't think I feel super strongly either way.

Reply:
I think that Quality Assurance testing will reveal what properties of the AWS service would make the bot run best (as well as how to price it). The dev can certainly provide insight if there is only supposed to be 1 of the bots in use, or if more can be supported. But ultimately, it's a QA function to determine what the maximum number will be for stability, and an external dev can only hint at what they think might be best.